spec/system/apo_spec.rb

```diff
@@ -50,12 +51,12 @@
     expect(page).to have_css('.disabled', text: 'Edit APO')
     expect(page).to have_css('.disabled', text: 'Create Collection')
 
-    # Manually complete the accessionWF steps to allow the object to be openable.
-    accession_step_count.times do
-      # Ensure every step of accessionWF is completed, this will allow us to open a new version.
-      click_link 'accessionWF'
-      click_button 'Set to completed', match: :first
+    # Ensure every step of accessionWF is completed, this will allow us to open a new version.
+    druid = page.current_url.split('/').last
+    accession_processes.each do |process|
+      workflow_client.update_status(druid:, workflow: 'accessionWF', process:, status: 'completed')
     end
+    click_link 'Reindex'
```

Comment:
these changes in the tests are needed because we can no longer click the "Set to completed" button in the UI for all steps - so just do it via the workflow client (same change in a different spec below)